### DISCUSSION TITLE
Cannot start a new codespace?

### DISCUSSION THREAD

**Discussion by minsikshin** [community/community#20918] [Codespaces]
Whenever I open a new codespace on a repo, it build the image and the container, but fails to connect to it. It shows me a screen to restart and reconnect to the codespace, but it gives me the same page after a while. The error details say 
"VSCodeWorkbench mount failed with "script "https://main.vscode-cdn.net/stable/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/loader.js" failed to load.""

Any suggestions?

**Comment by thaisonzdx** [comment]
You can try this:

1. Open **Command Palete** by press `F1` or click `Setting` button on left corner
2. Type "Kill VS Code..." and select **Codespaces: Kill VS Code Server on Host...

**Reply by minsikshin** [reply]
The problem is I can't access the Codespace at all so I can't run commands. I've been deleting the codespace in the 'manage codespace' tab if that's another method.

**Reply by jfullstackdev** [reply]
I was just using it a while ago, and created 3x I think so, no problems encountered, maybe it's on your side like connection, memory, extensions .. etc.

**Reply by minsikshin** [reply]
It seems like Codespaces will only work in VS Code Desktop for me, so I guess that will be my solution for the time being. Thanks for the help

**Comment by jjaseh** [comment]
Hi, 
I am having the same problem and this is the only result I have found on the web. After using VSCode cloud version for weeks it crashed with this same error detail. 
Have you found any other solution besides working on the desktop version?
Thank you!
![Capture](https://user-images.githubusercontent.com/109236207/181204712-d2bab4d5-277a-4ada-b07f-e2e9b9b44746.PNG)
